.github/workflows/dashboards-issue-add-label.yml

```diff
@@ -22,7 +21,7 @@ jobs:
         id: check
         shell: bash
         run: |
-          if [ -n "${{ (secrets.ISSUE_COMMANDS_TOKEN != '') || '' }}" ]; then
+          if [ "${{ github.repository }}" == "grafana/grafana" ] && [ -n "${{ secrets.GRAFANA_MISC_STATS_API_KEY }}" ]; then
```

Comment:
do you really need `GRAFANA_MISC_STATS_API_KEY`?

Reply:
No, I don't. And yeah, that sounds good. I just removed the entire job. Thank you!